### DISCUSSION TITLE
DeepL翻译服务无法连接

### DISCUSSION THREAD

**Discussion by DreammStill** [immersive-translate/immersive-translate#1935] [Help]
### 概述 | Summary

不知道是我的网络问题还是服务器问题，我无法连接DeepL服务，提示Failed to fetch，我有API密钥

**Comment by theowenyoung** [comment]
是插件还是油猴？

可以尝试切换 代理 或者关掉代理试试。

**Reply by DreammStill** [reply]
是微软扩展，没有开启代理
